<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainwindowClass</class>
 <widget class="QMainWindow" name="MainwindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>568</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>831</width>
    <height>568</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mainwindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="fileChoose">
    <property name="geometry">
     <rect>
      <x>711</x>
      <y>50</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="showArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>531</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>679</width>
       <height>529</height>
      </rect>
     </property>
     <widget class="QTextBrowser" name="codePreviewer">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>631</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Hash">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>334</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hash查找</string>
      </property>
     </widget>
     <widget class="QLabel" name="Binary">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>334</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>二分查找</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>340</y>
        <width>16</width>
        <height>181</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="fileDir">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>4</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>文件路径：</string>
      </property>
     </widget>
     <widget class="QLabel" name="showFileDir">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>5</y>
        <width>561</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>----</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="hashResultViewer">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>301</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="binaryResultViewer">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>360</y>
        <width>301</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="startSearch">
    <property name="geometry">
     <rect>
      <x>711</x>
      <y>160</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始查找</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
